dev-requirements.txt

```diff
@@ -2,8 +2,10 @@
 # TODO: how to automate switching from develop to version branches?
 git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
 git+https://github.com/dbt-labs/dbt-common.git
-git+https://github.com/dbt-labs/dbt-adapters.git
+git+https://github.com/dbt-labs/dbt-adapters.git@mcknight/ct-2819
+# -e /Users/matthewmcknight/git/dbt-adapters
 git+https://github.com/dbt-labs/dbt-adapters.git#subdirectory=dbt-tests-adapter
```

Comment:
You need to update the branch for `dbt-tests-adapter` too, to pick up the new test case.